form.js

```diff
@@ -0,0 +1,173 @@
+let a = false;
+let b = false;
+let c = false;
+let d = false;
+let e = false;
+let f = false;
+document.getElementById("checkoutbutton").disabled = true;
+function validateName() {
+    document.getElementById("checkoutbutton").disabled = true;
+    if ((document.myForm.firstname.value.length) < 4) {
+        document.myForm.firstname.style.borderColor = "red";
+        (document.getElementById("fname").previousElementSibling.innerHTML) = `<i class="fa fa-user"></i> Name` + " " + "[Name must contain minimum 4 letters!]";
```

Comment:
It is considered good manners to move big hard-coded strings into separate const to make your solution more readable and reusable.
```suggestion
const invalidNameMessage = `<i class="fa fa-user"></i> Name` + " " + "[Name must contain minimum 4 letters!]"; 

(document.getElementById("fname").previousElementSibling.innerHTML) = invalidNameMessage;
```